jsonargparse_tests/test_core.py

```diff
@@ -333,7 +333,12 @@ def test_parse_path_file_not_readable(parser, tmp_cwd):
     config_path = Path("config.json")
     config_path.touch()
     config_path.chmod(0)
-    pytest.raises(TypeError, lambda: parser.parse_path(config_path))
+    try:
+        config_path.read_text()
+    except PermissionError:
+        pytest.raises(TypeError, lambda: parser.parse_path(config_path))
+    else:
+        pytest.skip("File was still readable after chmod(0)")
```

Comment:
Please skip the entire test function using a decorator, instead of this try-except block.